definitions/features/pulp2.rb

```diff
@@ -5,7 +5,9 @@ class Features::Pulp < ForemanMaintain::Feature
     label :pulp2
 
     confine do
-      find_package('pulp-server') && !check_min_version('katello-common', '4.0')
+      !check_min_version('katello-common', '4.0') &&
+        ForemanMaintain::Utils::Service::Systemd.new('pulp_resource_manager', 0).exist? &&
```

Comment:
@jlsherrill the pulp2 should be available if service is enabled? This ensures once we complete the migration to pulpcore only pulpcore is available, specifically on 6.9.

Reply:
+1 this makes sense